reference/regions.html.markerb

```diff
@@ -14,51 +14,67 @@ Run <code>fly platform regions</code> to get a list of regions.
+| iad       | Ashburn, Virginia (US)        | ✓           |                                                       |
+| jnb       | Johannesburg, South Africa    |             |                                                       |
+| lax       | Los Angeles, California (US)  | ✓           |                                                       |
+| lhr       | London, United Kingdom        | ✓           |                                                       |
+| mad       | Madrid, Spain                 |             |                                                       |
+| mia       | Miami, Florida (US)           |             |                                                       |
+| nrt       | Tokyo, Japan                  | ✓           |                                                       |
+| ord       | Chicago, Illinois (US)        | ✓           |                                                       |
+| otp       | Bucharest, Romania            |             |                                                       |
+| phx       | Phoenix, Arizona (US)         |             |                                                       |
+| qro       | Querétaro, Mexico             |             |                                                       |
+| scl       | Santiago, Chile               | ✓           |                                                       |
+| sea       | Seattle, Washington (US)      | ✓           |                                                       |
+| sin       | Singapore, Singapore          | ✓           |                                                       |
+| sjc       | San Jose, California (US)     | ✓           |                                                       |
+| syd       | Sydney, Australia             | ✓           |                                                       |
+| waw       | Warsaw, Poland                |             |                                                       |
+| yul       | Montreal, Canada              |             |                                                       |
+| yyz       | Toronto, Canada               | ✓           |                                                       |
+
+- **Gateway regions:** "Gateway" regions also have WireGuard gateways, through which you connect to your organization's private network.
+- **Launch Plan or higher regions:** For some higher-demand regions, we restrict scaling up Machines to organizations with the [Launch, Scale, or Enterprise plans](https://fly.io/plans).
+
+## Edge-only regions
+
+Edge servers receive inbound traffic from the Internet destined for apps on the Fly.io platform and private traffic between Machines in different regions. [Fly Proxy](/docs/reference/fly-proxy/) routes traffic from edges to regional worker servers.
+
+We have a few regions that don't have workers and aren't available for app deployment:
```

Comment:
```suggestion
Edge servers receive inbound traffic from the Internet destined for apps on the Fly.io platform and private traffic between Machines in different regions. [Fly Proxy](/docs/reference/fly-proxy/) routes traffic from edges to regional worker servers. Edge-only regions don't have any workers and therefore aren't available for app deployment.
```

Small suggestion to move that last sentence into the paragraph above - it felt a bit like a non sequitur as a separate sentence.